README.md

```diff
@@ -1,17 +1,78 @@
+
+- A hidden, yet accessible Selenium2 `driver` instance, which allows you to focus
+on the *application under test* (AUT) instead of Selenium2 implementation details.
+- Easy parameterization, normalization, and setting of testing variables, like the AUT's host,
+the browser type to test with, [Sauce Labs](https://saucelabs.com/) integration, timeouts 
+for locating injected content after page-load etc. 
+- Convenient helper functions like `find_element`, `find_elements` that take [locators](http://rtomac.github.io/robotframework-selenium2library/doc/Selenium2Library.html#Locating%20elements)
+*or* [WebElements](http://selenium-python.readthedocs.org/en/latest/api.html#module-selenium.webdriver.remote.webelement) as parameters.
+- Built-in and readable *assertion methods* on the page object. For example `location_should_be`, `title_should_be` etc.
+- Much more...
+
+Like we said, the package is very flexible: it doesn't force you to use Robot, nor forces you
+to do heavy page object modeling up front. This is great for convicing your organization to move toward
+BDD and page object development because you can approach those ideals iteratively. Even
+with light modeling, and use of a non-BDD framework, your test suites
+can still benefit from the above listed features. Here's an example of a very minimally
+abstracted page object, where we're using a page object assertion (`title_should_be`) 
+outside of the Robot framework. `mytest.py`  
+
+    from robotpageobjects import Page
+    from unittest import TestCase
 
-The main point of using page objects is to factor out page implementation details (locators, UI details etc.) from the actual test suites. This makes the tests read more about the services a page offers and what's being tested instead of the internals of the page. It also makes your tests much more maintainable. For example, if a developer changes an element ID, you only need make that change once--in the appropriate page object.
 
-Each page object is simply a Robot library that inherits from this package's base `Page` class. These library classes can work independently of Robot
-Framework, even though they ultimately inherit from Robot Framework's Selenium2Library. This  allows you to encapsulate page logic Robot libraries, but use those libraries in any testing framework, including 
-Python [unittest](https://docs.python.org/2/library/unittest.html) test cases.
+    class MyPage(Page):
+        uri = "/some/path"
+        selectors = {
+            "the portlet": "xpath://some/complicated/xpath"
+        }
 
+    class MyTestCase(TestCase):
+        def setUp(self):
+            self.page = Page()
+            self.page.open()
 
-## Installing
+        def tearDown(self):
+            self.page.close()
 
-    $ pip install robotframework-pageobjects
+        def test_title(self):
+            self.page.title_should_be("My Page")
+
+        def test_portlet_renders(self):
```

Comment:
This method isn't used in the example.  I think your point would be clearer if it is.